web/admin/store/store.js

```diff
@@ -17,7 +18,13 @@ export function AdminStoreProvider({ children }) {
     employments: [],
     companies: [],
     missions: [],
-    minWorkDaysCursor: []
+    minWorkDaysCursor: [],
+    activitiesSettings: {
+      period: "day",
+      users: [],
+      maxDate: isoFormatLocalDate(new Date(Date.now())),
+      minDate: null
```

Comment:
Perhaps you could user *isoFormatLocalDate(getStartOfMonth(now()))* to initialize minDate ?

Reply:
I didn't want to write the logic (minDate = start of month) at two different places ...